src/status_im2/navigation/screens.cljs

```diff
@@ -66,6 +66,14 @@
                                            :to       0
                                            :duration constants/onboarding-modal-animation-duration}}}}})
 
+(def new-to-status-modal-animations
+  {:showModal    {:translationX {:from     (:width (rn/get-window))
+                                 :to       0
+                                 :duration constants/onboarding-modal-animation-duration}}
+   :dismissModal {:translationX {:from     0
+                                 :to       (:width (rn/get-window))
+                                 :duration constants/onboarding-modal-animation-duration}}})
```

Comment:
For :new-to-status screen we now use a modal, but change transition animations to behave like a normal push. This is because we need to preserve the screen behind as a background and the only way to do that is using a modal with `:modalPresentationStyle` option with `:overCurrentContext` as its value